packages/common/src/core/schemas/internal/metrics/StatCardUiSchema.ts

```diff
@@ -8,7 +8,6 @@ import { UiSchemaRuleEffects, IUiSchema } from "../../types";
  * @returns
  */
 export const buildUiSchema = (
```

Comment:
I'm just wondering if we should still leave `i18nScope` as an arg but just not use it anywhere in the `uiSchema` for now 🤔 I'm fine either way though. I can't really think of a use-case where these strings would change based on the context in which this editor is used.